tools/enumerate-devices/rs-enumerate-devices.cpp

```diff
@@ -195,19 +195,21 @@ int main(int argc, char** argv) try
     SwitchArg show_options_arg("o", "option", "Show all the supported options per subdevice");
     SwitchArg show_calibration_data_arg( "c", "calib_data", "Show extrinsic and intrinsic of all subdevices" );
     SwitchArg show_defaults("d", "defaults", "Show the default streams configuration");
-    SwitchArg only_sw_arg( "", "sw-only", "Show only software devices (playback, dds, etc. -- but not USB/HID/etc.)" );
+    SwitchArg only_sw_arg( "", "sw-only", "Show only software devices (playback, DDS, etc. -- but not USB/HID/etc.)" );
+    SwitchArg basic_formats_arg( "", "basic-formats", "Don't show non-raw conversions" );
```

Comment:
As discussed today, this will evolve into something more than a bool.
Will it be in this PR or a future one?

Reply:
Future.